stdlib/LinearAlgebra/src/triangular.jl

```diff
@@ -272,6 +272,11 @@ Base.isstored(A::UpperTriangular, i::Int, j::Int) =
 @propagate_inbounds getindex(A::UpperTriangular, i::Integer, j::Integer) =
     i <= j ? A.data[i,j] : _zero(A.data,j,i)
 
+Base._reverse(A::UpperTriangular, dims::Integer) = (reverse((Matrix(A)); dims))
```

Comment:
```suggestion
Base._reverse(A::UpperTriangular, dims::Integer) = reverse(Matrix(A); dims)
```
The extra parens are not needed.